cluster/charts/rook-ceph/values.yaml

```diff
@@ -244,18 +247,30 @@ csi:
   forceCephFSKernelClient: true
   #rbdLivenessMetricsPort: 9080
   #kubeletDirPath: /var/lib/kubelet
-  #cephcsi:
-    #image: quay.io/cephcsi/cephcsi:v3.1.2
-  #registrar:
-    #image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1
-  #provisioner:
-    #image: k8s.gcr.io/sig-storage/csi-provisioner:v2.0.0
-  #snapshotter:
-    #image: k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.0
-  #attacher:
-    #image: k8s.gcr.io/sig-storage/csi-attacher:v3.0.0
-  #resizer:
-    #image: k8s.gcr.io/sig-storage/csi-resizer:v1.0.0
+  cephcsi:
+    image: # in image.registry.quay
```

Comment:
The problem I see here is that the comments are not always preserved, and a user will not know what registry goes along with this image if the comment is missing. I would suggest also making `image:registry` an option, even though most users will not need to change it.

At the same time, I suggest that maybe `registry` be renamed to `registries`, and allow `registries:` to be added/removed as desired. The default could be as Travis suggested above.
```
registries:
  rook: docker.io
  sigstorage: k8s.gcr.io
  cephcsi: quay.io
``` 

But I could add my own registry if I wanted to change it more specifically:
```
registries:
  myregistry: my.company.net
  sigstorage: k8s.gcr.io
  cephcsi: my.company.net
```

And I could change images like:
```
cephcsi:
  image:
    registry: cephcsi
    repository: cephcsi/cephcsi
    tag: v3.1.2
```